official/docs/python/batches/add_shipments_to_batch.py

```diff
@@ -0,0 +1,10 @@
+import easypost
+
+
+easypost.api_key = "EASYPOST_API_KEY"
+
+batch = easypost.Batch.retrieve("batch_...")
+
+batch.add_shipments(shipments=[{"id": "shp_..."}, {"id": "shp_..."}])
```

Comment:
If we add a comma here then run `make format-python`, it will break these up on separate lines automatically.

```suggestion
batch.add_shipments(shipments=[{"id": "shp_..."}, {"id": "shp_..."},])
```